src/proc.c

```diff
@@ -51,12 +51,11 @@ extern int opt_proc_mem;
 extern int opt_use_waitpid_timeout;
 extern int opt_verbose;
 
-int proc_stopped(int pid)
+char proc_state(int pid)
 {
     FILE *f;
     char buf[128];
-    char c;
-    int rc = -1;
+    char c = -1;
```

Comment:
Well... I don't like it.
Maybe better return `int` like e.g. `fgetc()` ?

Reply:
Ok, let's make it consistent with standard C library functions. Fixup: https://github.com/tbricks/tbstack/pull/14/commits/95153c8e260fbb4542f8049f72e6eb77b0206e90. Note that the difference between `fgetc()` and this method is that `(char) -1` is valid return value for `fgetc()` but not for `proc_state()`